CHANGES.rst

```diff
@@ -6,8 +6,22 @@ Changes
 Unreleased
 ==========
 
+Breaking Changes
+----------------
+
+**IMPORTANT**: You must update your Sphinx project's ``Makefile`` to pick up
+these changes. See ``docs/Makefile`` for details.
+
```

Comment:
Not true. Changes from #88 are only in `rules.mk`. They will be automatically pulled when bumping the version within the `build.json` file within the downstream repository, maybe after one more incantation of `make reset`.